app/services/object_file_validator.rb

```diff
@@ -108,6 +108,6 @@ def object_client
 
   # @return [Boolean]
   def using_file_manifest?
-    file_manifest&.exists?
+    batch.using_file_manifest
   end
```

Comment:
**refactor:** just inspect the database model object itself to decide if we are using a file manifest instead of using that janky exists? method on the file manifest object